src/mpi/mpi_caf.c

```diff
@@ -3285,17 +3327,17 @@ co_reduce_1 (MPI_Op op, gfc_descriptor_t *source, int result_image, int *stat,
   if (rank == 0 || PREFIX (is_contiguous) (source))
     {
       if (result_image == 0)
-        ierr = MPI_Allreduce (MPI_IN_PLACE, source->base_addr, size, datatype,
+	ierr = MPI_Allreduce (MPI_IN_PLACE, source->base_addr, size, datatype,
```

Comment:
please revert whitespace only changes by amending the commit if you know how... if not, I'll do it.